tox.ini

```
@@ -104,6 +104,7 @@ setenv=
     LC_ALL=en_US.UTF-8
     WAGTAIL_SHARING_HOSTNAME=content.localhost
     DJANGO_SETTINGS_MODULE=cfgov.settings.test
+    TEST_DATABASE_URL={env:TEST_DATABASE_URL:postgres://cfpb:cfpb@localhost/cfgov_test}
```

```suggestion
    TEST_DATABASE_URL={env:TEST_DATABASE_URL:postgres://cfpb:cfpb@localhost/cfgov}
```

Using the `_test` suffix here sets the default Django database to `cfgov_test` but then it creates a test database named `test_cfgov_test`. This could just be `cfgov` which would use a test database name of `test_cfgov`.

More generally, though, would it be better to instead remove this line, add `DATABASE_URL` to `passenv` a few lines up (replacing `TEST_DATABASE_URL` which is already there), and have Travis/GHA just use `DATABASE_URL`? Then we'd be consistent everywhere:

1. For local development, tests run against whatever your `DATABASE_URL` is, consistent with your local server.
2. For Travis/GHA, tests also run against `DATABASE_URL`.

Thinking about this more, this would then remove the need to maintain the `TEST_DATABASE_URL` special case entirely, and perhaps we could totally eliminate that variable. This would make the workflow: if you want to run tests against another database, just set `DATABASE_URL`, e.g. `DATABASE_URL=foo tox`. I'm not sure it's worth keeping the complexity of the separate `TEST_DATABASE_URL` in this case.

---

Just getting caught up... and I agree, I like your ideas to simplify!

---

@chosak @willbarton I've removed references to `TEST_DATABASE_URL` so that we rely on `DATABASE_URL` for everything.